src/safe_apps/tests/test_views.py

```diff
@@ -267,6 +267,17 @@ def test_apps_returned_on_non_existent_client_url(self) -> None:
         self.assertEqual(response.status_code, 200)
         self.assertCountEqual(response.json(), json_response)
 
+    def test_apps_returned_on_null_url(self) -> None:
+        url = reverse("v1:safe-apps:list") + "?clientUrl=\0"
+
+        response = self.client.get(path=url, data=None, format="json")
+        self.assertEqual(response.status_code, 200)
+
+        url = reverse("v1:safe-apps:list") + "?url=\0"
+
+        response = self.client.get(path=url, data=None, format="json")
+        self.assertEqual(response.status_code, 200)
+
```

Comment:
Nitpick: I would just split this in two tests (one for `clientUrl` and another one for `url`) – so that we get independent feedback on the status of both checks.

Reply:
Good suggestion, done